<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZebraTRACE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow_rc.qrc">
    <normaloff>:/image/images/zebra.png</normaloff>:/image/images/zebra.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>9</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelPreview">
         <property name="text">
          <string>Preview:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxPreview">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Large Preview</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wireframe Overlay</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelTransparency">
         <property name="text">
          <string>Transparency:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="SliderTransparency">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>289</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>comboBoxPreview</zorder>
      <zorder>SliderTransparency</zorder>
      <zorder>horizontalSpacer_2</zorder>
      <zorder>labelPreview</zorder>
      <zorder>labelTransparency</zorder>
      <zorder>horizontalSpacer</zorder>
      <zorder>horizontalSpacer_3</zorder>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutContainer"/>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>0</height>
         </size>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <widget class="QWidget" name="tabOptions">
         <attribute name="title">
          <string>Options</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>251</width>
            <height>51</height>
           </rect>
          </property>
          <property name="title">
           <string>Image Size:</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>231</width>
             <height>22</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>W:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxX">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999999</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>1024</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>H:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QSpinBox" name="spinBoxY">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999999</number>
              </property>
              <property name="value">
               <number>1024</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>251</width>
            <height>141</height>
           </rect>
          </property>
          <property name="title">
           <string> Curves:</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>231</width>
             <height>111</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Number of Curves (n):</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxCurves">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>80</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_1">
              <property name="text">
               <string>Resolution:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxResolution">
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.300000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Width Min:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxMin">
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Width Max:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxMax">
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>251</width>
            <height>131</height>
           </rect>
          </property>
          <property name="title">
           <string>Function for Curves:</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>231</width>
             <height>100</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Range Min:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxAlphaMin">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Range Max:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxAlphaMax">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>-1000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>6.283000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Function X</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEditX">
              <property name="text">
               <string>(0.95+0.02*sin(20*a))*i/n</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Function Y:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEditY">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tabColors">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Colors</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tabInfo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Info</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonTrace">
            <property name="text">
             <string>Trace</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSaveSVG">
            <property name="text">
             <string>Save SVG</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonQuit">
            <property name="text">
             <string>Quit</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>pushButtonSaveSVG</zorder>
         <zorder>pushButtonQuit</zorder>
         <zorder>pushButtonTrace</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>progressBar</zorder>
   <zorder>frame</zorder>
   <zorder>frame_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenBitmap"/>
    <addaction name="actionSaveSVG"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionBackground"/>
    <addaction name="actionBorder"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpenBitmap">
   <property name="icon">
    <iconset>
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open Bitmap...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSaveSVG">
   <property name="text">
    <string>Save SVG...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionBackground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Background</string>
   </property>
  </action>
  <action name="actionBorder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Border &amp;Frame</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>spinBoxX</tabstop>
  <tabstop>spinBoxY</tabstop>
  <tabstop>spinBoxCurves</tabstop>
  <tabstop>doubleSpinBoxResolution</tabstop>
  <tabstop>doubleSpinBoxMin</tabstop>
  <tabstop>doubleSpinBoxMax</tabstop>
  <tabstop>doubleSpinBoxAlphaMin</tabstop>
  <tabstop>doubleSpinBoxAlphaMax</tabstop>
  <tabstop>lineEditX</tabstop>
  <tabstop>lineEditY</tabstop>
  <tabstop>pushButtonTrace</tabstop>
  <tabstop>pushButtonSaveSVG</tabstop>
  <tabstop>pushButtonQuit</tabstop>
  <tabstop>comboBoxPreview</tabstop>
  <tabstop>SliderTransparency</tabstop>
 </tabstops>
 <resources>
  <include location="mainwindow_rc.qrc"/>
 </resources>
 <connections/>
</ui>
